unit2/unit2.ipynb

```diff
@@ -1284,10 +1284,10 @@
     {
       "cell_type": "code",
       "source": [
-        "push_to_hub(repo_id=\"ThomasSimonini/q-FrozenLake-v1-8x8-slippery\", \n",
-        "                model=model,\n",
-        "                env=env\n",
-        "                )"
+        "repo_id = ",
```

Comment:
Maybe something like this would avoid confusions...WDYT?

```py
username = "" # FILL THIS
repo_name = "q-FrozenLake-v1-noSlippery1"
push_to_hub(
    repo_id=f"{username}/{repo_name}", 
    model=model,
    env=env
)
```

Reply:
Yep, definitely sounds better. Will do!

Reply:
Done!